src/scene-config.ts

```diff
@@ -50,7 +50,7 @@ const sceneConfig = {
         dampingFactor: 0.2,
         minPolarAngle: 0,
         maxPolarAngle: 2.8,
-        minZoom: 0.01,
+        minZoom: 0.0001,
```

Comment:
think this might be too small tbh. once you get to this level of zoom your camera becomes immobile. we need better controls.

Reply:
Ok yep, this was unintentional, and only used in my testing